### DISCUSSION TITLE
Connection line with dynamic handles

### DISCUSSION THREAD

**Discussion by azamat7g** [bcakmakoglu/vue-flow#764] [Q&A]
Hi! I created node with dynamic handles. If I create dynamic handle and connect to this handle node after that if I delete handle connected line jumps to another handle. If delete all dynamic handles connected line jumps to main handle. Your library has some kind of function that fix this issue or i have fix it manually?

Example: https://codesandbox.io/p/sandbox/jovial-buck-fsuh41

<video src="https://user-images.githubusercontent.com/10195180/225908295-f2177ccd-bd01-47a6-a79f-a31ea7f13837.mov" data-canonical-src="https://user-images.githubusercontent.com/10195180/225908295-f2177ccd-bd01-47a6-a79f-a31ea7f13837.mov" controls="controls" muted="muted"></video>

**Comment by bcakmakoglu** [comment]
I can't see the video but the behavior you describe is the desired outcome.
Edges *cannot lead nowhere* - they have to lead to a handle.
You *can* hide handles with opacity or something but you *cannot* remove a handle and expect the edge to "stick" where the handle used to be.
